Makefile

```diff
@@ -16,9 +13,9 @@ endif
 check-links: $(GET_LINK_CHECKER_IF_NEEDED)
 	$(HTMLTEST) $(HTMLTEST_ARGS)
 
-docker-serve:
-	docker run --rm -it -v $(PWD):/src -p 1313:1313 $(DOCKER_IMG) server $(DRAFT_ARGS)
-
 get-link-checker:
 	rm -Rf $(HTMLTEST_DIR)/bin
 	curl https://htmltest.wjdp.uk | bash -s -- -b $(HTMLTEST_DIR)/bin
+
+clean:
+  rm -rf $(HTMLTEST_DIR) public/* resources
```

Comment:
nit: should this be a tab to match the rest of the spacing in the MAKEFILE?

Reply:
Yep, just fixed that -- strange that I wasn't having any problems locally 🤷🏼‍♂️.